### PR TITLE
Draft

### PR DESCRIPTION
**Pull request by toddmotto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,28 @@
...
+	.config(function ($httpProvider) {
+        $httpProvider.useApplyAsync(true);
+	});
+```
+
+Here we're injecting `$httpProvider` (that configures everything to do with `$http`) and turning applyAsync on. But what does it actually do?
+
+applyAsync batches our `$http` calls that happen at a similar time into one digest cycle call. If we have two `$http` calls that happen at the same time, normally we'd be running two digest cycles - this isn't great for performance.
+
+Instead, applyAsync will notice that these two requests are near each other and wait for them both to complete, running one digest cycle after that. This is great! It saves a lot of processing time for one little configuration object.
```
Comment (ipc103) [inline README.md]: Sounds great - are there any tradeoffs to this? Any reason why I wouldn't want to turn this on?

**Comment by ipc103** [comment]
Nice short README. Can you add some context to the advantages/disadvantages to using this feature? Why wouldn't you want to do this? Likewise, how will performance improve by using this feature (page will load faster, etc.) Give the reader some more context to help them decide when to implement this.

**Comment by ipc103** [comment]
I'm thinking about adding a line to the end - "Improving performance is all about trade offs. As you build more Angular applications, you'll become comfortable deciding when to implement this feature." What do you think?

**Comment by toddmotto** [comment]
Sounds good, like it! :)

**Pull request merged**